thirdparty/README.md

```diff
@@ -731,3 +731,8 @@ Files extracted from upstream source:
 
 - lib/{common/,compress/,decompress/,zstd.h}
 - LICENSE
+
+## SMOLV
```

Comment:
+ I think it needs to be pushed between line 601 and line 602 of that file in order to keep alphabet order.